resources/views/courses/create.blade.php

```diff
@@ -11,36 +11,39 @@
 		<!-- Course name -->
 		<div class="field">
 			<label for="courseName">Course name</label>
-			<input type="text" name="courseName" id="courseName" required>
+			<input type="text" name="courseName" id="courseName">
 		</div>
 		<!-- Teacher(s) -->
-		<div class="field" title="Are you giving this course with another teacher?">
+		<div class="field">
 			<label for="teachers">Teacher(s)</label>
-			<select class="ui fluid search dropdown" name="teachers[]" id="teachers" multiple required>
+			<select class="ui fluid search dropdown" name="teachers[]" id="teachers" multiple>
 				@foreach ($teachers as $teacher)
-					<option value={{ $teacher['id'] }}> {{ $teacher['name'] }} </option>
+					<option value={{ $teacher['id'] }} {{ ($teacher['id'] == Auth::user()->id) ? 'selected' : '' }}> {{ $teacher['name'] }} </option>
```

Comment:
Nit: en un PR pasado, decidimos usar la arrow notación `$teacher->id` 

![image](https://user-images.githubusercontent.com/14140121/56484066-8c119780-6482-11e9-9b5e-7827f1834fcc.png)

Reply:
Double quote node attributes: (i.e. `value="..."`). Also if you're only getting the ID use `Auth::id()`